src/main/java/com/elytradev/concrete/inventory/gui/widget/Widget.java

```diff
@@ -33,9 +33,8 @@
 import net.minecraftforge.fml.relauncher.Side;
 import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class WWidget {
+public class Widget {
```

Comment:
No, JComponent is the root of all Swing components. A JComponent is also an AWT Container and Component for compatibility, as all Swing components can have children.